src/components/site/InputBox.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Card from 'react-md/lib/Cards/Card';
+import CardBody from 'react-md/lib/Cards/CardText';
+import TextField from 'react-md/lib/TextFields/TextField';
+import './StyleCard.css';
+
+class InputBox extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      redshiftVal1: 0,
+      redshiftVal2: 0,
+    };
+  }
+
+  handleChange = name => event => {
+    const { onUserInput } = this.props;
+    this.setState({
+      [name]: event,
+    });
+    onUserInput(name, event);
+  };
+
+  render() {
+    return (
+      <Card className="card">
+        <CardBody>
+          <h2>
+            Enter the range of redshift values you would like to see. <br />
+          </h2>
+          <div>
+            <span style={{ margin: '10px' }} className="inline">
+              From
+            </span>
+            <span>
+              <TextField
+                id="inline-text-input-1"
+                type="number"
+                style={{ maxWidth: '200px' }}
+                className="inline-block"
+                label=""
+                onChange={this.handleChange('redshiftVal1')}
+                lineDirection="center"
+                placeholder="Type Number Here"
+              />
+            </span>
+            <span style={{ margin: '10px' }} className="inline">
+              to
+            </span>
+            <span>
+              <TextField
+                id="inline-text-input-2"
+                type="number"
+                style={{ maxWidth: '200px' }}
+                className="inline-block"
+                label=""
+                onChange={this.handleChange('redshiftVal2')}
+                lineDirection="center"
+                placeholder="Type Number Here"
+              />
```

Comment:
Each of these `TextField`s are so similar, consider DRYing this up.  For instance you could create a function that returns a `TextField` wrapped in a `<span>` and takes an `id` and the redshift key string (i.e. `redshiftVal1`) as arguments.  That way, if you ever want to change their props, or styles, or whatever, you only ever have to do so in one place.  something like:
```
renderTextField(id, redshiftKey) {
   return(
      <span><TextField
         id={`inline-text-input-${id}`}
         type="number"
         style={{ maxWidth: '200px' }}
         className="inline-block"
         label=""
         onChange={this.handleChange(redshiftKey)}
         lineDirection="center"
         placeholder="Type Number Here"
      /></span>
   );
}
```

Or something like that.  Know what I mean?